tools/gh_api.py

```diff
@@ -66,13 +66,13 @@ def get_auth_token():
     response = requests.post('https://api.github.com/authorizations',
                             auth=(user, pw), data=json.dumps(auth_request))
     if response.status_code == 401 and \
-       response.headers.get('X-GitHub-OTP') == 'required; sms':
-        print("Your login API resquest a SMS one time password", file=sys.stderr)
-        sms_pw = getpass.getpass("SMS password: ", stream=sys.stderr)
+            'required;' in response.headers.get('X-GitHub-OTP'):
```

Comment:
What does `response.headers.get()` give if it's not there? A normal dict would give `None`, which will fail an `in` test.